index.js

```diff
@@ -152,10 +152,12 @@ module.exports = {
       // that version contains API to hook fastboot into ember-cli
 
       app.use((req, resp, next) => {
+        var fastbootQueryParam = (req.query.hasOwnProperty('fastboot') && req.query.fastboot === 'false') ? false : true;
+        var enableFastBootServe = !process.env.FASTBOOT_DISABLED && fastbootQueryParam;
```

Comment:
@mfazekas If you set `FASTBOOT_DISABLED`, irrespective of the query param you will never be served via fastboot. The environment variable is temporary and will go away after the double builds are resolved.